test/jsApi.yml

```diff
@@ -0,0 +1,130 @@
+- test: Test js input
+  steps:
+    -   in: bn create node8 returnInput
+    -   in: |-
+            echo 'exports.handler = (input) => ({ inputType: typeof input, input});' >function.js
+    -   in: bn deploy returnInput
+    -   in: curl -s "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"undefined"}'
+    -   in: curl -s --data '{}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"foo":"bar","yet":123}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{"foo":"bar","yet":123}}'
+    -   in: curl -s --data '["foo","bar"]' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":["foo","bar"]}'
+    -   in: bn remove returnInput
+- test: Test js method 
+  steps:
+    -   in: bn create node8 returnMethod
+    -   in: |-
+            echo 'exports.handler = (_, req) => req.method;' >function.js
+    -   in: bn deploy returnMethod
+    -   in: curl -s -X GET "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"GET"'
+#    -   in: curl -s -X HEAD "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
```

Comment:
If you place code in comments please explain in the comments why they contain code.

Reply:
removed for now. will put it back once it works